src/stories/CalendarCard.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import CalendarCard from "@/components/cards/CalendarCard";
+import type { StoryFn } from "@storybook/react";
+
+export default {
+  title: "Components/Cards/CalendarCard",
+  component: CalendarCard,
+};
+
+const Template: StoryFn = () => {
+  return <CalendarCard />;
+};
```

Comment:
StoryFn은 사용하지 않아도 괜찮을 것 같습니다!
```suggestion
const Template = () => <CalendarCard />;
```

Reply:
넵 수정하겠습니다!